### PR TITLE
migrate interfaces to python 3

### LINKED ISSUE #133: Migrate interfaces to Python 3
https://github.com/satanas/libturpial/tree/development/libturpial/lib/interfaces

This change should:
* Migrate the existing code to python 3
* Add more tests to cover functionality (if necessary)
* Ensure previous tests pass locally

### PR DESCRIPTION
**Pull request by categulario**
and their tests

### CODE REVIEW

File: libturpial/lib/services/url/shortypython/imports.py [hunk 1]
```diff
@@ -1,11 +1,7 @@
-## Shorty
-## Copyright 2009 Joshua Roesslein
-## See LICENSE
```
Comment (satanas) [inline libturpial/lib/services/url/shortypython/imports.py]: Please, don't remove the copyright.

**Review by satanas — COMMENTED** [review]
Everything looks good except for a file where the copyright is removed. Once you fix that I will merge the PR. Thanks!

File: libturpial/lib/services/url/shortypython/imports.py (hunk 1 above)
Comment (categulario) [reply libturpial/lib/services/url/shortypython/imports.py]: it got removed by the 2to3 tool. I'll restore it

**Comment by categulario** [comment]
I'm marking this as a draft because I don't think it makes sense to merge it now.

**Comment by categulario** [comment]
I see you changed the branch to python3, therefore it doesn't matter if this is not complete, you may merge

**Comment by satanas** [comment]
Sounds good! Merging 🚢

**Pull request merged**